lib/galaxy/tools/parameters/basic.py

```diff
@@ -2134,9 +2137,11 @@ def __init__(self, tool, input_source, trans=None):
         super().__init__(tool, input_source, trans)
         self._parse_formats(trans, input_source)
         collection_types = input_source.get("collection_type", None)
+        tag = input_source.get("tag")
```

Comment:
This need to be described in the tool schema, and should probably be a list too ?
A test showing how that's used in a tool would be good too.